src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.html

```diff
@@ -7,38 +7,54 @@
                 is-elevate-customer={isElevateCustomer}
                 onbatchdryrun={handleBatchDryRun}
                 onedit={handleEditBatch}
+                is-batch-processing={isBatchProcessing}
                 onprocessbatch={handleProcessBatch}>
         </c-ge-batch-gift-entry-header>
     </template>
 
-    <c-ge-form-renderer
-            batch-id={recordId}
-            onsubmit={handleSubmit}
-            onsectionsretrieved={handleSectionsRetrieved}
-            onpermissionerror={handlePermissionErrors}
-            ontogglemodal={handleToggleModal}
-            onnavigate={handleNavigateEvent}
-            loading-text={loadingText}
-            batch-currency-iso-code={batchCurrencyIsoCode}>
-    </c-ge-form-renderer>
+    <div if:true={isBatchProcessing} class='spinner-container'>
```

Comment:
Not a blocker. I prefer having these `if:true` directives on their own `template` ala `<template if:true={isBatchProcessing}>` personally. I think it's easier to tell at a glance which elements are included in the conditional rendering instead of having the directive on an element that may also have classes and other attributes.

Reply:
Updated!